app/generator/content/schedule/part1/index.js

```diff
@@ -11,11 +11,10 @@ const part1 = (schedule) => {
     stack: [
       { image: `${imagePath}/rpa-logo.jfif`, fit: [millimetresToPoints(200), millimetresToPoints(25)], style: 'scheduleLogo' },
       getAddress(schedule.businessName, schedule.address),
-      { text: `${schedule.scheme.name}`, style: 'header1' },
-      { text: 'Revised payment schedule', style: 'subTitle' },
-      getBusinessName(schedule.businessName),
       getSBI(schedule.sbi),
-      getAgreementReference(schedule.scheme.agreementNumber)
+      getBusinessName(schedule.businessName),
+      getAgreementReference(schedule.scheme.agreementNumber),
```

Comment:
This displays wrong title should be `Agreement number:` but is `Agreement reference number:`

Also, we have 2 of the same content for both statement (`app/generator/content/statement/part1/get-agreement-number.js`) and schedule (`app/generator/content/schedule/part1/get-agreement-reference.js`), can we move this into the common area like `app/generator/content/get-business-name.js` and reference it in both cases.

Reply:
`Agreement reference number` now changed to `Agreement number` in schedule, therefore content of the 2 files are no more the same.